virttest/qemu_storage.py

```diff
@@ -48,7 +48,9 @@ def filename_to_file_opts(filename):
                     file_opts['user'] = matches.group('user')
     elif filename.startswith('rbd:'):
         filename_pattern = re.compile(
-            r'rbd:(?P<pool>.+?)/(?P<image>[^:]+)(:conf=(?P<conf>.+))?')
+            r'rbd:(?P<pool>.+?)/((?P<namespace>.+?)/(?P<img>[^:]+)|(?P<image>[^:]+))'
```

Comment:
```suggestion
            r'rbd:(?P<pool>.+?)/(?P<namespace>.+?(?=/))?/?(?P<image>[^:]+)(:conf=(?P<conf>.+))?'
```
it might not add an alias `img` for `image`.
```

>>> filename_pattern = re.compile(r'rbd:(?P<pool>.+?)/(?P<namespace>.+?(?=/))?/?(?P<image>[^:]+)(:conf=(?P<conf>.+))?')
>>> filename_pattern.match('rbd:rbd/czx/test.img').groupdict()
{'pool': 'rbd', 'namespace': 'czx', 'image': 'test.img', 'conf': None}
>>> filename_pattern.match('rbd:rbd/test.img').groupdict()
{'pool': 'rbd', 'namespace': None, 'image': 'test.img', 'conf': None}
```